symfony/framework-bundle/3.3/config/services.yaml

```diff
@@ -1,3 +1,6 @@
+# This file is the entry point to configure your own services.
+# Files in the packages/ subdirectory configure your dependencies.
```

Comment:
Yes and no. You can store your own configuration under `packages/` as well. There is no restrictions. People are free to organize their config as they see fit (same for the routing).